static/js/apps/eval_rig/data_store.tsx

```diff
@@ -34,10 +35,48 @@ import {
   DC_STAT_FEEDBACK_COL,
   LLM_STAT_FEEDBACK_COL,
   QUERY_ID_COL,
+  QUERY_OVERALL_FEEDBACK_COL,
   USER_COL,
 } from "./constants";
 import { Response } from "./types";
 
+// Gets the document path for a specific query or call
+export function getPath(
+  sheetId: string,
+  queryId?: number,
+  callId?: number
+): string[] {
+  const path = ["sheets", sheetId];
+  if (queryId) {
+    path.push(...["queries", String(queryId)]);
```

Comment:
in what condition queryId and/or callId is not set?

Reply:
this is for getting the path to a document in firestore, so if we want to get the path to a query document, we wouldn't set a callId

Reply:
should queryId always be set?